D/D.sublime-syntax

```diff
@@ -697,16 +700,20 @@ contexts:
       scope: punctuation.section.parens.end.d
       pop: true
   pragma-attribute:
+    - match: '\b({{pragmas}})\b'
+      scope: variable.function.d constant.other.d
```

Comment:
Uncertain about `msg|printf|scanf|` but the others rather look like plain flags or something along those lines. Sure those still should be scoped `variable.function`?

Using `variable.function` in general looks suspicious, but I am probably not familiar enough with D to judge it.

Reply:
Look up the D specification. Also variable.function was already there

Reply:
A am aware of `variable.function` already having been used. According to D specification, I'd judge said `msg`, `printf` and `scanf` pragmas possible functions, but the others just appear to be some sort of compiler flags. Thus `variable.function` doesn't seem appropriate - semantically.


In general current D syntax is somewhat lazy with regards to scope names - sometimes even doesn't strictly follow some of the official guidelines.